LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs

```diff
@@ -1599,6 +1600,32 @@ private static void GetIteConfigurationsB(ISuperIO superIO, Manufacturer manufac
 
                         break;
 
+                    case Model.X670E_Valkyrie: //IT8625E
+                        v.Add(new Voltage("CPU Core Voltage", 0));
+                        v.Add(new Voltage("CPU DDR IMC Voltage", 1));
+                        v.Add(new Voltage("+12.0V", 2, 10, 2));
+                        // Voltage of unknown use
+                        v.Add(new Voltage("Voltage #4", 3, true));
+                        v.Add(new Voltage("CPU_MISC Voltage", 4));
+                        v.Add(new Voltage("CPU_VDD Voltage", 5));
+                        v.Add(new Voltage("CPU_SOC Voltage", 6));
```

Comment:
Thanks for this PR! Please follow naming scheme as elsewhere in this file, e.g. +12V, CPU SoC, CPU Optional Fan (below), etc.

Reply:
@PhyxionNL 
Fixed in e6588e0

Reply:
Thanks, merged!